document/core/syntax/modules.rst

```diff
@@ -231,17 +241,19 @@ The |MELEM| component of a module defines a vector of *element segments* that in
 .. math::
    \begin{array}{llll}
    \production{element segment} & \elem &::=&
-     \{ \ETABLE~\tableidx, \EOFFSET~\expr, \EINIT~\vec(\funcidx) \} \\
+     \{ \ETABLE~\tableidx, \EOFFSET~\expr, \EINIT~\vec(\funcidx), \EACTIVENESS~\activeness \} \\
```

Comment:
I think a passive segment should have neither a table index nor an offset. So may be make these different records. Then you can leave the "activeness" flag implicit as well.